test/models/billing_profile_test.rb

```diff
@@ -102,4 +102,13 @@ def test_create_default_for_user_creates_a_billing_profile
     assert_equal(billing_profile.country_code, user_without_billing_profile.country_code)
     assert_equal(billing_profile.name, user_without_billing_profile.display_name)
   end
+
+  def test_billing_address_is_mirrored_to_linked_invoice
+    # Invoice which is linked with @billing_profile
```

Comment:
How about embodying this knowledge in the code? like `invoice_with_billing_profile`, which becomes a private method of `BillingProfileTest`?